apis/projectcontour/v1alpha1/contourdeployment.go

```diff
@@ -114,6 +114,10 @@ type ContourSettings struct {
 	// Deployment describes the settings for running contour as a `Deployment`.
 	// +optional
 	Deployment *DeploymentSettings `json:"deployment,omitempty"`
+
+	// WatchNamespaces restricts watching contour instances to a subset of namespaces.
```

Comment:
I think we want the documentation here and the release note to state that this field is for limiting the namespaces the *Contour instance* will watch for resource updates (and not what the Gateway Provisioner will watch)